bin/git-post-receive-hook-puppet

```diff
@@ -9,6 +9,9 @@ export MY_REPO=ssh://git.mysociety.org/data/git/private/mysociety-puppet.git
 # Reset environment so we can use git
 unset GIT_INDEX_FILE
 
+# Set a sane umask for shared directories
```

Comment:
Fair point about squashing the commit - I should have done that.

I understand that he script is run in the context of the user making the push. It's clear that some users' umasks are set to 0022 on this host (my user's was by default, and that seems to be the Debian default - see https://wiki.debian.org/Debate/umask). Explicitly setting this here ensures the correct value, but you're right, it's probably better explained in the comment -  we can't make any assumptions about how a user will have set their umask.